src/patternfly/components/ExpandableSection/expandable-section.scss

```
@@ -23,6 +18,12 @@
   --#{$expandable-section}--m-expanded__toggle-icon--Rotate: 90deg;
   --#{$expandable-section}--m-expanded__toggle-icon--m-expand-top--Rotate: -90deg;
   
+  // Toggle text
+  --#{$expandable-section}__toggle-text--Color: var(--pf-t--global--color--brand--default);
+  --#{$expandable-section}__toggle-text--hover--Color: var(--pf-t--global--color--brand--hover);
+  --#{$expandable-section}__toggle--m-expanded__toggle-text--Color: var(--pf-t--global--color--brand--hover);
```

Just a small nit, `pf-m-expanded` applies to `$expandable-section/.pf-v5-c-expandable-section`, so the `--m-expanded` bit would go on that element in the var name. Also no need for `__toggle` in there.

```suggestion
  --#{$expandable-section}--m-expanded__toggle-text--Color: var(--pf-t--global--color--brand--hover);
```